Stp.TestingApi/EnumMapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Stp.TestingApi.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Stp.TestingApi
```

Comment:
Sorry, I have planned to use this for testing. But this is not necessary. I`ll delete this.